.github/actions/setup-env/action.yml

```diff
@@ -62,7 +62,7 @@ runs:
       run: |
         pip install \
           git+https://github.com/bokeh/bokeh-fastapi.git@main \
-          git+https://github.com/holoviz/panel@7377c9e99bef0d32cbc65e94e908e365211f4421
+          git+https://github.com/holoviz/panel@fastapi_serve
```

Comment:
This implicitly updates to `panel==1.5.*`. Thus, we'll have to check styling as well.